packages/frontend/src/components/connect-button/styles.module.css

```diff
@@ -7,6 +7,26 @@
 }
 
 .wrapper {
+    @apply flex gap-3;
+}
+
+.networkWrapper {
+    @apply bg-white w-10 h-10 rounded-md flex gap-3 items-center justify-center;
+}
+
+.networkIcon {
+    @apply w-7 h-7;
+}
+
+.networkRow {
+    @apply h-10 flex items-center px-2 gap-2 hover:bg-gray-200 transition-colors;
```

Comment:
We should add the `duration-200` class here, for the transition.

Reply:
Addressed in 72a15a7b69529c090da6887d86c5a5f2d3fb8b08